stan/math/prim/meta/require_helpers.hpp

```diff
@@ -308,7 +308,7 @@ using container_type_check_base
   template <template <class...> class TypeCheck, class... Check>               \
   using require_##check_type##_vt = require_t<                                 \
       container_type_check_base<checker, value_type_t, TypeCheck, Check...>>;  \
-  /*! \brief Require type does not satisfy checker and */                      \
+  /*! \brief Require type does not satisfy checker or */                       \
   /*! value type satisfies `TypeCheck` */                                      \
```

Comment:
Should there be an extra not on the value_type here?

Reply:
I'm not following where you are looking at?

Reply:
"Require type does not satisfy check or value type satisfies `TypeCheck`"

vs. the other which is

"Require type does not satisfy check or scalar type *does not* satisfy `TypeCheck`"

And the difference in the name is the vt vs. st and that doesn't seem to imply any more or less nots.

Reply:
Ah whups good call!